client/src/components/graphs/OrganizationalChart.tsx

```diff
@@ -189,260 +151,608 @@ const OrganizationalChart = ({
-          xmin: Math.min(box.xmin, nodeArg.x ?? 0),
-          xmax: Math.max(box.xmax, nodeArg.x ?? 0),
-          ymin: Math.min(box.ymin, nodeArg.y ?? 0),
-          ymax: Math.max(box.ymax, nodeArg.y ?? 0)
-        }),
-        {
-          xmin: Number.MAX_SAFE_INTEGER,
-          xmax: Number.MIN_SAFE_INTEGER,
-          ymin: Number.MAX_SAFE_INTEGER,
-          ymax: Number.MIN_SAFE_INTEGER
+
+    // Wait for the next tick to ensure DOM updates
+    await new Promise(resolve => setTimeout(resolve, 50))
+    const dataUrl = await toPng(chartRef.current, {
+      backgroundColor: BACKGROUND_COLOR,
+      filter: node => {
+        if (controlsRef.current?.contains(node)) {
+          return false
         }
-      )
-      return {
-        box: boundingBox,
-        size: [
-          boundingBox.xmax - boundingBox.xmin + nodeSize[0],
-          boundingBox.ymax - boundingBox.ymin + nodeSize[1]
-        ],
-        center: [
-          (boundingBox.xmax + boundingBox.xmin + nodeSize[0] - 200) / 2,
-          (boundingBox.ymax + boundingBox.ymin + nodeSize[1] - 50) / 2
-        ]
+        return true
       }
-    }
+    })
 
-    tree.current.nodeSize(nodeSize)
-    const bounds = calculateBounds(root)
-    const scale = Math.min(
-      1.2,
-      1 / Math.max(bounds.size[0] / width, bounds.size[1] / height)
+    const link = document.createElement("a")
+    link.download = exportTitle + ".png"
+    link.href = dataUrl
+    link.click()
+  }
+
+  const filterPeople = people => {
+    return people
+      .filter(position => {
+        if (!position.person) {
+          return false
+        }
+        if (peopleFilter === "none") {
+          return false
+        }
+        if (peopleFilter === "leaders") {
+          return position.role === "LEADER"
+        }
+        if (peopleFilter === "leaders_deputies") {
+          return position.role === "LEADER" || position.role === "DEPUTY"
+        }
+        return true
+      })
+      .sort((a, b) => {
+        return rolePriority[a.role] - rolePriority[b.role]
+      })
+      .map(position => position.person)
+      .slice(0, peopleLimits[peopleFilter] ?? undefined)
+  }
+
+  const determineSymbol = (org, allAscendantOrgs) => {
+    const ascendantOrgs =
+      utils
+        .getAscendantObjectsAsList(org, allAscendantOrgs, "parentOrg")
+        ?.reverse() || []
+    const context = utils.determineApp6field(ascendantOrgs, "app6context", "0")
+    const standardIdentity = utils.determineApp6field(
+      ascendantOrgs,
+      "app6standardIdentity",
+      "1"
     )
-    canvas.attr(
-      "transform",
-      `translate(${width / 2 - scale * bounds.center[0]},${
-        height / 2 - scale * bounds.center[1]
-      }) scale(${scale})`
+    const symbolSet = utils.determineApp6field(
+      ascendantOrgs,
+      "app6symbolSet",
+      "00"
+    )
+    const hq = org?.app6hq || "0"
+    const amplifier = org?.app6amplifier || "00"
+    const version = "14" // APP-6E
```

Comment:
Please check latest version of `OrganizationalChart.tsx` on **main**: we switched back to APP-6D!